c7n/resources/lakeformation.py

```diff
@@ -0,0 +1,30 @@
+from c7n.manager import resources
+from c7n import query
+from c7n.query import QueryResourceManager, ConfigSource, TypeInfo
+from c7n.tags import universal_augment
+
+
+class DescribeResource(query.DescribeSource):
+    def augment(self, resources):
+        return universal_augment(
+            self.manager,
+            super(DescribeResource, self).augment(resources))
+
+
+@resources.register('lakeformation')
+class LakeFormation(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'lakeformation'
+        enum_spec = ('list_resources', 'ResourceInfoList[].ResourceArn', None)
+        arn = id = 'ResourceArn'
+        name = 'name'
+        detail_spec = ("describe_resource", "ResourceArn", None, "ResourceInfo")
```

Comment:
Do we need `detail_spec` here? It doesn't seem to return any information we're not getting from `list_resources`.

Reply:
@ajkerrigan : Thank you for reviewing ! I have removed detail_spec .

Reply:
Great, thanks! 👍